packages/components/src/radio-group/types.ts

```diff
@@ -0,0 +1,13 @@
+export type RadioGroupProps = {
+	label: string;
+	checked?: string | number;
+	defaultChecked?: string | number;
+	disabled?: boolean;
+	onChange?: ( value: string | number | undefined ) => void;
+	children: React.ReactNode;
+};
+
+export type RadioProps = {
+	value: string | number;
+	children?: React.ReactNode;
+};
```

Comment:
Let's add JSDocs comments the each prop here — we can use the README as a starting point.